CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.1.0]
+### Added
+- Add PublicInterface type ([#197](https://github.com/MetaMask/utils/pull/197))
```

Comment:
nit:
```suggestion
- Add `PublicInterface` type ([#197](https://github.com/MetaMask/utils/pull/197))
```

Reply:
Thanks, updated! I prefer using backticks when referencing elements of the package API as well.